src/gluonts/mx/block/quantile_output.py

```diff
@@ -20,13 +20,46 @@
 from gluonts.mx import Tensor
 
 
+def uniform_weights(x: list) -> List[float]:
```

Comment:
Can we add a more concrete type?

And let's try to avoid single letter variables. Especially `x` should not be a list.

Reply:
> Can we add a more concrete type?

We could, but it would be an artificial restriction in my opinion. In this context `x: List[float]` but could be a list of anything

Reply:
Right. Could we then maybe add a docstring explaining what this does and have a quick example like:

```
>>> uniform_weights(["a", "b", "c", "d"])
[0.25, 0.25, 0.25, 0.25]
```